libs/@local/hash-backend-utils/src/hash-instance.ts

```diff
@@ -134,41 +134,22 @@ export const isUserHashInstanceAdmin = async (
     });
```

Comment:
Can simplify `isUserHashInstanceAdmin` above now? We can fetch the teamId directly then get the relations

Reply:
Makes sense, thank you